dist/backdraft.js

```diff
@@ -845,11 +845,19 @@ _.extend(Plugin.factory, {
     ',
     sFilterListTemplate : '\
       <li><label><input class="list" id="value" type="checkbox" name="{{= attr }}" \
-         value="{{= options }}" /> <%= options %></label></li> \
+         value="{{= options }}" /> {{= options }}</label></li> \
+    ',
+    sActionButtonTemplate : '\
+      <button class="btn btn-primary btn-sm {{= klass}}" name="button" type="submit" title="">{{= caption}}</button>\
```

Comment:
Nesting sounds awesome.  Will do.  I think the Hungarian notation there is just because I didn't want a variable conflict and wanted to clarify the difference between the base string used to make the template and the template itself.  (e.g. ``sFilterStringTemplate`` and ``filterStringTemplate``)  Is there a more preferred naming to do to tell the difference between these, or would you be happy with just the nesting?

Will use className.  Much better.  :)